src/adaptors/tcp_adaptor.c

```diff
@@ -567,7 +567,7 @@ static void handle_connection_event(pn_event_t *e, qd_server_t *qd_server, void
         int read = handle_incoming(conn);
         conn->last_in_time = tcp_adaptor->core->uptime_ticks;
         conn->bytes_in += read;
-        qd_log(log, QD_LOG_DEBUG, "[C%"PRIu64"] PN_RAW_CONNECTION_READ Read %i bytes", conn->conn_id, read);
+        qd_log(log, QD_LOG_DEBUG, "[C%"PRIu64"] PN_RAW_CONNECTION_READ Read %i of %"PRIu64" bytes", conn->conn_id, read, conn->bytes_in);
```

Comment:
This is a very helpful improvement. My only comment is that the wording isn't immediately obvious. Perhaps "Read %i bytes. Total read %"PRIu64" bytes"?

Reply:
Agreed. Will do.